libp2p/Host.cpp

```diff
@@ -767,7 +767,7 @@ void Host::startedWorking()
         // time, stop will wait on m_timer and graceful network shutdown.
         Guard l(x_runTimer);
         // create deadline timer
-        m_timer.reset(new boost::asio::deadline_timer(m_ioService));
+        m_timer.reset(new ba::deadline_timer(m_ioService));
```

Comment:
Do we need heap allocated timers? They cannot be used as values?

Reply:
It looks we destroy and recreate it when the network is stopped / restarted, but I'm not sure if that's actually needed.

Reply:
I'd like to leave the timer alone for these changes (they were meant to be trivial changes with absolutely no potential for functional impact), I'd be happy to look at it later.